src/styles/index.ts

```diff
@@ -1639,7 +1639,7 @@ const styles = (theme: ThemeColors) =>
 
         popoverInnerContainer: {
             paddingTop: 0, // adjusting this because the mobile modal adds additional padding that we don't need for our layout
-            maxHeight: '95%',
+            maxHeight: '100%',
```

Comment:
Please check past PR where 95% value was added and confirm that this doesn't cause any regression which that PR fixes.